Sources/Commands/SwiftTestTool.swift

```diff
@@ -230,6 +233,8 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                     diagnostics.emit(data: SpecifierDeprecatedDiagnostic())
                 }
 
+                try fileManager.removeFileTree(try self.buildParameters().buildPath.appending(component: "codecov"))
```

Comment:
This can be removed, the directory will be already deleted before the switch case.

Reply:
done